src/Auth0Client.ts

```diff
@@ -1161,7 +1148,6 @@ export class Auth0Client {
     options.authorizationParams = options.authorizationParams || {};
 
     options.authorizationParams.scope = getUniqueScopes(
-      this.defaultScope,
```

Comment:
Is this intentional? It looks like this will not include the defaultScope's anymore? Previously, `this.scope` and `this.options.authorizationParams?.scope` where identical, Wouldnt that mean we now need to replace

```
this.defaultScope,
this.optoins.authorizationParams
```
with
```
this.scope
```

Reply:
Maybe we can also avoid the assignment here and use

```
const scope = getUniqueScopes(...)
```
I believe we want to avoid mutating it like we were doing already.

We would then also need to use `scope` when constructing the `CacheKey`.

Reply:
Yeah my bad I should have moved to `this.scope`, will add a test to cover this case and then fix

Reply:
Updated this to use `this.scope` and moved to assigning `options` similar to how we do elsewhere. There were a bunch of `options.authorzationParams.scope` references so figured I'd keep the changes minimal